src/OpenTelemetry/Metrics/MetricPoint.cs

```diff
@@ -326,7 +326,11 @@ internal void Update(double number)
                         lock (this.histogramBuckets.LockObject)
                         {
                             this.runningValue.AsLong++;
-                            this.histogramBuckets.RunningSum += number;
+                            if (number > 0)
```

Comment:
I have different understanding - negative value for histogram should be dropped:

```csharp
if (number < 0)
{
    // do nothing for now, revisit once histogram supports negative
}
else
{
    lock (this.histogramBuckets.LockObject)
    {
        // do the work
    }
}

Reply:
Given the current wording is "CAN", I think either way is _okay_, I personally would prefer the conservative approach (do nothing).

https://github.com/open-telemetry/opentelemetry-proto/blob/f8c59fe6b063397714034d9727f08aca1a5b99b3/opentelemetry/proto/metrics/v1/metrics.proto#L406-L410